english/love-git.md

```diff
@@ -0,0 +1,3 @@
+I am in love with git and github since Udacity introduced me to it.
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.